src/health-tm/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "health-tm",
+  "version": "1.0.0",
+  "description": "Returns the current status and other information regarding the Tasking Manager",
+  "main": "health-tm-lambda.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "Andria Hibe <andriacohibe@gmail.com>",
+  "license": "ISC",
+  "dependencies": {
+    "node-fetch": "^2.6.0"
+  }
```

Comment:
is cloudfriend not required in each of the lambda JSONs as well?

Reply:
If you have it installed on the root package.json then it already works. :)

Reply:
ah, because it's a dev dependency, and only required for the deployment. Gotcha. :+1: